packages/preset-create-react-app/src/index.ts

```diff
@@ -89,6 +89,16 @@ const webpack = (
       }
     : {};
 
+  const plugins = craWebpackConfig.resolve.plugins.map(
+    (plugin: { appSrcs: string[] }) => {
+      if (plugin.appSrcs) {
+        // eslint-disable-next-line no-param-reassign
```

Comment:
Can you comment on this line? why are we doing this?

Reply:
Done!